docs/docs/using-mdx.mdx

```diff
@@ -209,7 +209,8 @@ console.log(<Example name="Mars" year={2022} />)
 ### Components
 
 There is one special prop: `components`.
-It takes an object mapping component names to components.
+It takes an object mapping component names to components, which MDX uses to
+replace default components with custom components.
```

Comment:
There are no default components. 🤔 
Perhaps wording like:

> It takes an object mapping component names to components. MDX adds the component names to the scope of the document to allow the component to be used.

Reply:
I was looking to use a verb like "replace" or "override", to:
1. be explicit that it is transforming something (I'm ok with changing the "something" of "default components" - I agree that it's not good)
2. use terminology closer to what community members have done

Reply:
You want to add one concrete sentence here.
I don’t think human language is expressive enough for that.
I think it needs to be illustrated.

there are no default components; a “custom” component implies non-custum components, what are those?
when you mention scope, I think it needs to be explained too, what does scope mean in MDX; it’s also not not *really* what happens.

> it is transforming something

It is *not* always transforming something: see the following lines. `Planet` is used and `Planet` is passed. That isn’t a transform.

> use terminology closer to what community members have done

Do you have sources for these?

Reply:
sources:

- https://github.com/mdx-js/mdx/issues/1385
- https://github.com/kentcdodds/mdx-bundler/issues/160
- https://github.com/mdx-js/mdx/issues/821
- i'm guessing there's more, with further research